traffic_stats/influxdb_tools/create/create_ts_databases.go

```diff
@@ -20,45 +20,81 @@ under the License.
 package main
 
 import (
-	"flag"
 	"fmt"
-	"os"
 
 	influx "github.com/influxdata/influxdb/client/v2"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
```

Comment:
`pkg/errors` actually seems pretty interesting. I saw it presented at the last GopherCon I think. The `cli` package seems to have some uses, but it looks like you aren't using them in the final form of this patch. I'd leave it out until it is necessary.

Both of these programs probably seem scripty because they were intended for limited use. One is for a one time setup of influx and I think the other was made for upgrades. They may have even once been written in Perl.